client.go

```diff
@@ -44,7 +44,7 @@ func New(apiKey string) *Client { return NewWithClient(apiKey, newDefaultHTTPCli
 // this, it is ill-advised to have the key directly in your source code.
 func NewWithClient(apiKey string, c *http.Client) *Client {
 	return &Client{
-		baseURL: "https://api.climacell.co/v3/",
+		baseURL: "https://api.test.climacell.co/v3/",
```

Comment:
Is `api.test.climacell.co` a new base URL? I tried a cURL request to there, and I got that I can't consume that service